components/dependency_injection/definitions.rst

```diff
@@ -4,6 +4,9 @@
 Working with Container Service Definitions
 ==========================================
 
+Service definitions are the instructions followed by the container to build a
```

Comment:
Maybe "[...] are the instructions describing how the container should build a service. They are not the actual services used by our applications" ?